payjoin/src/send/error.rs

```diff
@@ -269,19 +269,23 @@ impl From<InternalCreateRequestError> for CreateRequestError {
     fn from(value: InternalCreateRequestError) -> Self { CreateRequestError(value) }
 }
 
-/// Represent an error returned by the receiver.
+/// Represent an error returned by Payjoin receiver.
 pub enum ResponseError {
-    /// `WellKnown` errors following the BIP78 spec
-    /// https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki#user-content-Receivers_well_known_errors
-    /// These errors are displayed to end users.
+    /// Errors that are defined in the [`BIP78`] spec.
     ///
-    /// The `WellKnownError` represents `errorCode` and `message`.
+    /// It is safe to display these errors to end users.
+    ///
+    /// [`BIP78`]: https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki#user-content-Receivers_well_known_errors
```

Comment:
I prefer that the original explicitly said "`WellKnown` errors". There are lots of errors defined in the bip78 spec, these are explicitly called "Well-known" errors

I do also prefer your wording "It is safe to display these errors to end users."

Reply:
fixed. 
I was looking into the spec and could not find another errors, could  you be please share a link to errors beside the `ReceiverWellKnown` ?